frontend/src/scenes/userLogic.ts

```diff
@@ -76,6 +77,20 @@ export const userLogic = kea<userLogicType>([
                         return values.user
                     }
                 },
+                deleteUser: async () => {
+                    return await api
+                        .delete('api/users/@me/')
+                        .then(() => {
+                            actions.deleteUserSuccess(null)
```

Comment:
nit: Because this is a `loader` you dont need these manual actions. They will automatically get called if the async function resolves or throws. then you could move the actions.logout to the `deleteUserSuccess` listener which would be cleaner